resources/config.test.edn

```diff
@@ -85,4 +85,5 @@
                                                 :value-deserializer-class-config "org.apache.kafka.common.serialization.ByteArrayDeserializer"}}
             :tracer               {:enabled         [true :bool]
                                    :custom-provider ""}
-            :new-relic            {:report-errors false}}}
+            :new-relic            {:report-errors false}
+            :log-format           "text"}}
```

Comment:
default log format in test should be json?

Reply:
json wouldn't be required in tests. It isn't very readable

Reply:
I would say use default log format as json

Reply:
Use `"test"` as `:text` ?